### ISSUE TITLE
Minimizing the game window breaks menu crosshair

### ISSUE THREAD

**Issue by fayer3** [ferriarnus/VivecraftForge#198]
as in title, minimizing the game window makes the menu crosshair not work at all.

**Comment by Techjar** [comment]
Yeah this was a known issue with the standalone version too. For some reason minimizing the window causes it to have zero dimensions, breaking the mouse input.
